en/docs/learn/enterprise-marketing-software.md

```diff
@@ -0,0 +1,134 @@
+# Enterprise Marketing Software Integration
+
+If your business uses or is willing to use enterprise marketing softwares such as Hubspot or Salesforce, WSO2 IAM solution has the ability to interact with those systems while giving you the simplicity to manage your organizations&#39; identities from a single place.
+
+This topic provides instructions on how to integrate HubSpot with WSO2 Identity Server to provision users from WSO2 Identity Server. After completing this tutorial you can see the users you add using WSO2 Identity Server being created in HubSpot and lead information updates in WSO2 Identity Server being updated in HubSpot as well.
+
+## Configuring HubSpot
+
+1. **Test accounts are created and managed within developer accounts.** If you don&#39;t have a developer account in HubSpot, you can sign up from [here](https://app.hubspot.com/signup/developers?_ga=2.39153443.1802613489.1576611619-500942594.1573763828).
```

Comment:
add a new line before starting the statemet 'If you don&#39;t have a developer account..'

Reply:
Fixed https://github.com/wso2/docs-is/pull/1758/commits/154049c67d80d52fb96b2c1bc8ea421436b12aae